doc/user/content/release-notes.md

```diff
@@ -19,6 +19,12 @@ the PR description instead. They will be migrated here during the release
 process by the release notes team.
 {{< /comment >}}
 
+{{% version-header v0.25.0 %}}
+
+- Improve `base64` decoding to support numerals in encoded strings. Previously, if the encoded string contained a numeral, decoding would fail.
```

Comment:
The existing code was just completely busted, so I might say "fix" rather than "improve":

```suggestion
- Fix `base64` decoding to support numerals in encoded strings. Previously, if the encoded string contained a numeral, decoding would fail.
```